alibi_detect/saving/tensorflow/_saving.py

```diff
@@ -77,14 +77,14 @@ def save_model_config(model: Callable,
 
     if model is not None:
         filepath = base_path.joinpath(local_path)
-        save_model(model, filepath=filepath, save_dir='model')
+        save_model(model, filepath=filepath.joinpath('model'))
         cfg_model = {'src': local_path.joinpath('model')}
     return cfg_model, cfg_embed
 
 
 def save_model(model: tf.keras.Model,
                filepath: Union[str, os.PathLike],
-               save_dir: Union[str, os.PathLike] = 'model',
+               filename: str = 'model',
```

Comment:
Saving must also be updated since `save_model` was saving using an incorrect naming convention. i.e. everything was saved to `model.h5` instead of allowing `encoder.h5` etc